cmd/utils/flags.go

```diff
@@ -524,6 +525,16 @@ var (
 		Name:  "preload",
 		Usage: "Comma separated list of JavaScript files to preload into the console",
 	}
+	APIFilterGetLogsDeadlineFlag = cli.DurationFlag{
+		Name:  "api.filter.getLogs.deadline",
+		Usage: "Execution deadline for log collecting filter APIs (Default: 10s)",
+		Value: filters.GetLogsDeadline,
+	}
+	APIFilterGetLogsMaxItemsFlag = cli.IntFlag{
+		Name:  "api.filter.getLogs.maxitems",
+		Usage: "Maximum allowed number of return items for log collecting filter API (Default: 10000)",
```

Comment:
Isn't "Default" message printed if `Value` is set?

Reply:
I can understand the question. Does "message" mean log message or console message?

Reply:
Oh, I mean console message. 
Even default value is not mentioned in `Usage` field, console will automatically print the default value.

<img width="1228" alt="Screen Shot 2021-02-25 at 2 47 45 PM" src="https://user-images.githubusercontent.com/2115430/109109336-7523e680-7778-11eb-9245-43fe5faf12ae.png">
<img width="479" alt="Screen Shot 2021-02-25 at 2 47 51 PM" src="https://user-images.githubusercontent.com/2115430/109109345-7bb25e00-7778-11eb-88a6-6d2183aedea0.png">

Reply:
Oh, you're right. Thank you for the notice! I removed the default words.